components/common/listingCart/ListingCartMini.vue

```diff
@@ -6,7 +6,7 @@
           <div class="is-inline-flex is-align-items-center">
             <div>
               <b>{{ listingCartStore.count }}</b>
-              {{ $tc('listingCart.item', listingCartStore.count) }}
```

Comment:
why remove?
https://kazupon.github.io/vue-i18n/guide/pluralization.html#pluralization

Reply:
Thanks, for the link. I will fix. 👍

Reply:
$tc - `Locale message pluralization Supported for Legacy API mode only.`

Actually seems like in $tc has been deprecated and $t now supports pluralization, although the docs have not been updated. Since we do not use it anywhere else it's not worth enabling legacy mode and will be completely removed in future versions according to their Github comments.

![image](https://github.com/kodadot/nft-gallery/assets/22052693/1ca17240-2f93-48a2-9ea9-46d643eba57f)

Reply:
Alright noted thank you for this finding